bundles/org.openhab.binding.speedtest/src/main/resources/OH-INF/i18n/speedtest.properties

```diff
@@ -0,0 +1,57 @@
+# binding
+
+binding.speedtest.name = Speedtest Binding

```

Comment:
GitHub shows something at the end of the lines which could be a non-standard linefeed. Can you try to regenerate the file?

Reply:
fixed